src/linux/backend-hid.cpp

```diff
@@ -453,7 +453,7 @@ namespace librealsense
             }
         }
 
-        iio_hid_sensor::iio_hid_sensor(const std::string& device_path, uint32_t frequency)
+        iio_hid_sensor::iio_hid_sensor(const std::string& device_path, uint32_t frequency, float sensitivity)
```

Comment:
Sensitivity of what? gyro? accel?
We have 2 of those? 1 for each IMU sensor?

Reply:
Yes, the sensitivity is either gyro or in the future accel sensitivity, it is determined by hid_profile but both go through the same functions